### PR TITLE
Add a unit label to the progress bar

### PR DESCRIPTION
**Pull request by brianhelba**
This makes it display as "s/step" instead of "s/it".

### CODE REVIEW

File: nlisim/cli.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def run(obj, target_time: float) -> None:
 
     with tqdm(
         desc='Running simulation',
+        unit='step',
```
Comment (jbeezley) [inline nlisim/cli.py:42]: It will actually show simulation time in hours rather than time step.

Comment (brianhelba) [reply nlisim/cli.py:42]: Maybe I'm misunderstanding the semantics of simulation iteration. The README states:
> Now run simulation up to 50 timesteps using the first example config.

How should we refer to each full iteration of the simulation?

Comment (jbeezley) [reply nlisim/cli.py:42]: Yeah, the README isn't accurate.  The issue is there isn't necessarily a well defined time step globally.  See https://github.com/NutritionalLungImmunity/nlisim/pull/89.

Comment (brianhelba) [reply nlisim/cli.py:42]: Ok, for now, if we say something like
> Run the simulation for 50 __

What is the best description of "__"? "iterations"? "steps"? "timesteps"? "hours"?

Comment (jbeezley) [reply nlisim/cli.py:42]: My understanding is expressed in units of hours.

Comment (brianhelba) [reply nlisim/cli.py:42]: Ok, thanks. Closing this in favor of #98.

**Pull request closed without merging**